src/terrain/routes/schemas/vice.clj

```diff
@@ -46,12 +46,12 @@
      :command (describe [String] "The command used to start the analysis")}))
 
 (defschema ContainerStateWaiting
-  {:reason  (describe (maybe String) "The reason the container is in the waiting state")
-   :message (describe (maybe String) "The message associated with the waiting state")})
+  {:reason                 (describe (maybe String) "The reason the container is in the waiting state")
+   (optional-key :message) (describe (maybe String) "The message associated with the waiting state")})
```

Comment:
Probably yes, but just to double-check, is it intended that this has both `optional-key` and `maybe` used? (i.e., can it both be not present or null?)

Reply:
Technically possible, though unlikely. This stuff comes directly from the k8s API and I'm a bit uncertain about the possible combinations that are possible. I've already been bitten by it once, so I made it optional across the board to hopefully prevent a bunch of churn in PRs.